napari_arboretum/graph.py

```diff
@@ -1,22 +1,15 @@
-# ------------------------------------------------------------------------------
-# Name:     Arboretum
-# Purpose:  Dockable widget, and custom track visualization layers for Napari,
-#           to cell/object track data.
-#
-# Authors:  Alan R. Lowe (arl) a.lowe@ucl.ac.uk
-#
-# License:  See LICENSE.md
-#
-# Created:  01/05/2020
-# ------------------------------------------------------------------------------
+"""
+Classes and functions for working with graphs.
+
+Note that this file should *not* contain code for laying out the graphs for
+visualisation. Code for this is kept in `tree.py`.
+"""
```

Comment:
Honestly, not sure, but are doc strings the right thing to use here? Not that it matters, but what does PEP etc say?